modules/ossm-federation-features.adoc

```diff
@@ -6,7 +6,6 @@ This module included in the following assemblies:
 [id="ossm-federation-features_{context}"]
```

Comment:
Each .adoc file must contain a :_content-type: attribute in its metadata that indicates its file type. Please review all of the files in the PR and apply content type labels accordingly. I won't mention them all in this review. Please see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#content-type-attributes for more details.